project.md

```diff
@@ -0,0 +1,10 @@
+# Project docs
+
+## Release Process
+
+* Increment version in version.go
+* `git commit` version increment
+* `git tag` the prior commit (preferrably signed tag)
+* `make docs` to produce PDF and HTML copies of the spec
+* Make a release on [github.com/opencontainers/specs](https://github.com/opencontainers/specs/releases) for the version. Attach the produced docs.
```

Comment:
Yea, disregard the newline thing. Our team actually deprecated the one sentence per-line rule after finding that `git log -p --word-diff` and github's improved "smart diff" is sufficient: https://github.com/coreos/docs/blob/master/STYLE.md#line-wrapping-considered-harmful